tests/test_flask_api.py

```diff
@@ -39,6 +41,13 @@
 from anitya.lib.backends import REGEX
 from tests import Modeltests, create_distro, create_project, create_package
 
+# Py3 compatibility: UTF-8 decoding and JSON decoding become separate steps
+if sys.version_info[0] >= 3:
```

Comment:
I went with using the sys module mainly because this module already imports that. However, the suggestion to use `output.get_data(as_text=True")` as an inherently version independent alternative sounds more attractive, so I'll switch to that.